davidshare/Burger-Builder
# Add a NavBar for the app

#### What does this PR do?
Add the navbar for the app
#### Description of Task to be completed?
- create a component for each navigation item
- create a component for the navigation items
- create Toolbar component to host the main navigation and the logo
- add the navigation items to the Toolbar component
- add a logo component
- add an image for the burger-logo
- add styles for all the components created above
- install a file loader and configure webpack to use the loader
#### How should this be manually tested?
- clone the repository
- run npm install
- run npm start
- the navigation is displayed on the landing page
#### Any background context you want to provide?
- none
#### What are the relevant pivotal tracker stories?
- none
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/25446108/50653686-a9316180-0f8a-11e9-9878-94e6b778a888.png)

#### Questions:
- none
